CodeChef_problems/PPATTERN/PPATTERN.py

```diff
@@ -0,0 +1,27 @@
+# This loop will runs for each test case
+for x in range(1 , int(input()) + 1):
+    num = int(input()) # Taking the number of rows input
+    lis = []
+
+    # CORE LOGIC
+    # This for loop runs from (-ve num to +ve num)
+    for i in range(-num + 1 , num):
+        if num - abs(i) != num:
+            lis.append(num - abs(i))
+    lis.append(0)
+    print(lis)

```

Comment:
Remove `print(lis)` (line 12) since this will print unnecessary output

Reply:
@Vishruth-S I have removed the extra print statement